pkg/sql/sem/tree/show.go

```diff
@@ -1255,6 +1255,30 @@ func (node *ShowTenant) Format(ctx *FmtCtx) {
 	}
 }
 
+// ShowLogicalReplicationJobsOptions represents the WITH clause in SHOW LOGICAL REPLICATION JOBS.
+type ShowLogicalReplicationJobsOptions struct {
+	WithDetails bool
+}
+
+// ShowLogicalReplicationJobs represents a SHOW LOGICAL REPLICATION JOBS statement.
+type ShowLogicalReplicationJobs struct {
+	ShowLogicalReplicationJobsOptions
+}
+
+// Format implements the NodeFormatter interface.
+func (node *ShowLogicalReplicationJobs) Format(ctx *FmtCtx) {
+	ctx.WriteString("SHOW LOGICAL REPLICATION JOBS")
+
+	withs := []string{}
+	if node.WithDetails {
+		withs = append(withs, "DETAILS")
```

Comment:
nit: this could be simplified: could just add `ctx.WriteString(WITH DETAILS)` and remove `withs`